packages/opentelemetry-exporter-collector-grpc/README.md

```diff
@@ -32,48 +35,70 @@ provider.addSpanProcessor(new SimpleSpanProcessor(exporter));
 
 provider.register();
 
+Graceful.on("exit", async () => {
```

Comment:
can we just use https://nodejs.org/api/process.html#process_event_exit here?

Reply:
no, because ```Listener functions must only perform synchronous operations. The Node.js process will exit immediately after calling the 'exit' event listeners causing any additional work still queued in the event loop to be abandoned. In the following example, for instance, the timeout will never occur:```

Reply:
merging discussion to [thread](https://github.com/open-telemetry/opentelemetry-js/pull/2130#discussion_r617728668)